src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: start;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  display: block;
+  background: url(images/star.svg) no-repeat center center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+
+  --card-text: #060b35;
+  --card-additional-text: #616070;
+  --card-main-color: #00acdc;
+}
+
+.card__container {
+  padding: 16px;
+}
+
+.card__featured-image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 40px;
+  display: block;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--card-text);
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--card-additional-text);
+}
+
+.card__reviews {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__reviews-count {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 15px;
+  color: var(--card-text);
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.card__price-label {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--card-additional-text);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy-link {
+  width: 100%;
+  background: var(--card-main-color);
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 0;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  margin-top: 16px;
+}
+
+.card__buy-link:hover {
+  background: #fff;
+  border: 1px solid var(--card-main-color);
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover